cdap-security/src/main/java/co/cask/cdap/security/authorization/AuthorizerClassLoader.java

```diff
@@ -48,7 +50,14 @@ static ClassLoader createParent() {
       authorizerResources = ImmutableSet.of();
     }
 
-    final Set<String> finalAuthorizerResources = authorizerResources;
+    Set<String> apiResources;
+    try {
+      apiResources = ClassPathResources.getResourcesWithDependencies(baseClassLoader, Application.class);
```

Comment:
It would be good to add a comment saying cdap-api also includes Logger dependencies.